src/extension.ts

```diff
@@ -13,11 +15,50 @@ const clipboard = impor(
   './models/clipboard'
 ) as typeof import('./models/clipboard');
 
+const serialport = impor(
+  './models/serialportctrl'
+) as typeof import('./models/serialportctrl');
+
 // const serialport = impor(
 //   './models/serialport'
 // ) as typeof import('./models/serialport');
 
 export function activate(context: vscode.ExtensionContext) {
+  const serialportGetPlatform = vscode.commands.registerCommand(
+    'iotcube.serialportGetPlatform',
+    () => {
+      return serialport.SerialPortCtrl.getPlatform();
+    }
+  );
+
+  const serialportClose = vscode.commands.registerCommand(
+    'iotcube.serialportClose',
+    () => {
+      return serialport.SerialPortCtrl.close();
+    }
+  );
+
+  const serialportSend = vscode.commands.registerCommand(
+    'iotcube.serialportSend',
+    (payload: string) => {
+      return serialport.SerialPortCtrl.send(payload);
+    }
+  );
+
+  const serialportOpen = vscode.commands.registerCommand(
+    'iotcube.serialportOpen',
+    (comPort: string, option: PortOption) => {
+      return serialport.SerialPortCtrl.open(comPort, option);
+    }
+  );
+
+  const serialportChooseCOM = vscode.commands.registerCommand(
+    'iotcube.serialportChooseCOM',
+    async (board: Board | undefined) => {
```

Comment:
Use `board?: Board` for optional parameter

Reply:
Fixed